{"uuid": "bdfc8bb3-1a1d-4491-97c0-b80264c34100", "children": ["1cb7f370-37cd-43b6-b515-acccef40aa57"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55c5016aeffa <unknown>\n#1 0x55c50116d970 <unknown>\n#2 0x55c5011bf385 <unknown>\n#3 0x55c5011bf5b1 <unknown>\n#4 0x55c50120e3c4 <unknown>\n#5 0x55c5011e52bd <unknown>\n#6 0x55c50120b70c <unknown>\n#7 0x55c5011e5063 <unknown>\n#8 0x55c5011b1328 <unknown>\n#9 0x55c5011b2491 <unknown>\n#10 0x55c50167642b <unknown>\n#11 0x55c50167a2ec <unknown>\n#12 0x55c50165da22 <unknown>\n#13 0x55c50167ae64 <unknown>\n#14 0x55c501641bef <unknown>\n#15 0x55c50169d558 <unknown>\n#16 0x55c50169d736 <unknown>\n#17 0x55c5016ade76 <unknown>\n#18 0x7fcdf429caa4 <unknown>\n#19 0x7fcdf4329c3c <unknown>\n\n", "trace": "  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/QATEST/QATEST/src/tests/ui_tests/conftest.py\", line 30, in driver\n    WebDriverWait(driver, 10).until(\n  File \"/home/runner/work/QATEST/QATEST/venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1743002161064, "stop": 1743002172920}], "start": 1743002161064, "stop": 1743002172926}